servers/quarkus-server/src/main/resources/application.properties

```diff
@@ -128,7 +128,9 @@ quarkus.http.test-port=0
 quarkus.http.access-log.enabled=true
 # fixed at buildtime
 quarkus.resteasy.path=/api
-quarkus.resteasy.gzip.enabled=true
+quarkus.http.enable-compression=true
```

Comment:
According to [this page](https://quarkus.io/guides/resteasy#gzip-support):

> The configuration property `quarkus.http.enable-compression` has no effect on compression support of RESTEasy Classic endpoints.

So if I'm understanding this correctly, this change will effectively disable compression for all REST endpoints, but enable it for static resources. Is that the intent?

Reply:
Strange. IntelliJ shows that property as "not mapped". I've re-added it.